src/sql/src/plan/statement/show.rs

```diff
@@ -237,8 +237,13 @@ pub fn show_databases<'a>(
     scx: &'a StatementContext<'a>,
     filter: Option<ShowStatementFilter<Aug>>,
 ) -> Result<ShowSelect<'a>, PlanError> {
-    let query = "SELECT name FROM mz_catalog.mz_databases".to_string();
-    ShowSelect::new(scx, query, filter, None, Some(&["name"]))
+    let query = "SELECT name, COALESCE(comment, '') as comment
+         FROM mz_catalog.mz_databases databases
+         LEFT JOIN mz_internal.mz_comments comments
+           ON databases.id = comments.id
+         WHERE (comments.object_type = 'database' OR comments.object_type IS NULL)"
```

Comment:
Why do we need the `IS NULL` predicate?

Reply:
Ah because of the `LEFT JOIN` we don't want to filter out rows without comments, right?

Reply:
Exactly!